src/EnergyPlus/DataSurfaces.hh

```diff
@@ -679,6 +700,7 @@ namespace DataSurfaces {
         std::vector<int> ConstituentSurfaceNums; // A vector of surface numbers which reference this surface for representative calculations
         int ConstructionStoredInputValue;        // holds the original value for Construction per surface input
         SurfaceClass Class;
+        SurfaceClass OriginalClass;
```

Comment:
I probably need to move this from Surface to SurfaceWindow since only those cats have OriginalClass.